File: okhttplib/src/main/java/com/example/okhttplib/utils/GlideRequestOptionsUtils.java
package com.example.okhttplib.utils;

import com.bumptech.glide.request.RequestOptions;
import com.example.okhttplib.R;

/**
 * 默认配置图片glide  RequestOptions
 */
public class GlideRequestOptionsUtils {
    private static RequestOptions options;

    public static RequestOptions requestOptions() {
        if (options == null) {
            options = new RequestOptions()
                    .centerCrop()
                    .placeholder(R.drawable.placehold)
                    .error(R.drawable.placehold);
        }
        return options;
    }

}

File: okhttplib/src/main/java/com/example/okhttplib/utils/GlideRequestOptionsUtils.kt
package com.example.okhttplib.utils

import com.bumptech.glide.request.RequestOptions
import com.example.okhttplib.R

/**
 * 默认配置图片glide  RequestOptions
 */
object GlideRequestOptionsUtils {
    private var options: RequestOptions? = null

    fun requestOptions(): RequestOptions {
        if (options == null) {
            options = RequestOptions()
                    .centerCrop()
                    .placeholder(R.drawable.loading)
                    .error(R.drawable.loading)
        }
        return options as RequestOptions
    }

}
